bedrock/firefox/templates/firefox/dashboard-base.html

```diff
@@ -7,7 +7,7 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox-dashboard' %}
```

Comment:
Thinking the dashboard-base.html shouldn't change to page_css instead leave as an overwriting site_css. Otherwise the CSS load order gets wacky since the child templates have their own site overwrites like below in tiles.html

{% block site_css %}
  {{ super() }}
  {% stylesheet 'firefox-tiles' %}
{% endblock %}

Perhaps leaving interest-dashboard.html with a site_css block and the two child templates can change to the page_css block.

What do you think @jpetto?

Not really sure what this {{ super() }} does but if it doesn't affect if I use page_css or site_css block, I would think using page_css would be better for the child templates.